### PR TITLE
Add tlsEarlyData support to TargetHttpsProxy.

### PR DESCRIPTION
**Pull request by wu-bin**
Fixes https://github.com/hashicorp/terraform-provider-google/issues/18421

Tests that I run:
```
make testacc TEST=./google/services/compute TESTARGS='-run=TestAccComputeTargetHttpsProxy_'
```

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
compute: added `tlsEarlyData` field to `google_compute_target_https_proxy` resource
```

### CODE REVIEW

**Review by NickElliot — APPROVED** [review]
test failure is unrelated and persistent in nightly -- LGTM

**Review by NickElliot — CHANGES_REQUESTED** [review]
wait realized one thing, sorry about that

File: mmv1/products/compute/TargetHttpsProxy.yaml [hunk 1]
```diff
@@ -130,6 +130,18 @@ properties:
...
+    description: |
+      Specifies whether TLS 1.3 0-RTT Data (“Early Data”) should be accepted for this service.
+      Early Data allows a TLS resumption handshake to include the initial application payload
+      (a HTTP request) alongside the handshake, reducing the effective round trips to “zero”.
+      This applies to TLS 1.3 connections over TCP (HTTP/2) as well as over UDP (QUIC/h3).
+    values:
+      - :STRICT
+      - :PERMISSIVE
+      - :DISABLED
+    default_value: :DISABLED
```
[the remark this answers is not in the document]
Comment (Yashpandey4) [reply mmv1/products/compute/TargetHttpsProxy.yaml:144]: Ack, I can confirm that the server side default is automatically set to DISABLED. Using `default_from_api: true` makes sense here

[the remark this answers is not in the document]
Comment (wu-bin) [reply mmv1/products/compute/TargetHttpsProxy.yaml]: Thank you both! Changed to `default_from_api: true`.

**Review by NickElliot — APPROVED** [review]
LGTM :)

**Comment by wu-bin** [comment]
Thank you, @NickElliot !

**Pull request merged**